projects/dendrites/permutedMNIST/experiments/centroid.py

```diff
@@ -87,7 +91,7 @@ class CentroidFigure1BExperiment(CentroidFigure1B,
     batch_size=256,
     val_batch_size=512,
     epochs=3,
-    tasks_to_validate=[0, 1, 2, 3, 4, 9, 24, 49, 74, 99],
+    tasks_to_validate=list(range(10)),
```

Comment:
When setting `tasks_to_validate=list(range(10))` beware that the eval time grows quadratically with the num of tasks. That's we had it set to just a subset of tasks earlier. I would recommend reverting this change since we don't need to eval for every task.

Reply:
Good point about quadratic complexity. Cutting out those 4 validation points in the control flow (tasks 5-8) saves 30 validations. I set to all since I wanted to see the full time series of validation scores for each task, but that can happen once or for spot checking. Certainly doesn't need to be the default.